devito/passes/iet/mpi.py

```diff
@@ -368,3 +350,55 @@ def mpiize(graph, **kwargs):
         make_halo_exchanges(graph, mpimode=mpimode, **kwargs)
 
     make_reductions(graph, mpimode=mpimode, **kwargs)
+
+
+# *** Utilities
+
+def filter_iter_mapper(iet):
```

Comment:
Nitpick: if these aren't going to be exposed, do they need to be private? Either way, they should probably all be consistent with one another